### PR TITLE
Document passthrough render hooks

### PR DESCRIPTION
**Pull request by jmooring**
<https://deploy-preview-2668--gohugoio.netlify.app/render-hooks/passthrough/>

### CODE REVIEW

File: content/en/functions/transform/ToMath.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ action:
   aliases: []
   related:
     - content-management/mathematics
-  returnType: types.Result[template.HTML]
+  returnType: template.HTML
```
Comment (bep) [inline content/en/functions/transform/ToMath.md:10]: I think the old value was more correct.

File: content/en/render-hooks/passthrough.md [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+(`string`) The inner content of the passthrough element, excluding the delimiters.
+
+###### Ordinal
+
+(`int`) The zero-based ordinal of the passthrough element on the page.
+
+###### Page
+
+(`page`) A reference to the current page.
+
```
Comment (bep) [inline content/en/render-hooks/passthrough.md:85]: +PageInner

File: content/en/functions/transform/ToMath.md (hunk 1 above)
Comment (jmooring) [reply content/en/functions/transform/ToMath.md]: Fixed

File: content/en/render-hooks/passthrough.md (hunk 2 above)
Comment (jmooring) [reply content/en/render-hooks/passthrough.md:85]: Fixed

**Pull request merged**